scripts/unpinnedGitHubActionsImageVersions.fsx

```diff
@@ -4,35 +4,19 @@ open System
 open System.IO
 
 #load "../src/FileConventions/Library.fs"
-
-let NotInDir (dirName: string) (fileInfo: FileInfo) =
-    not(
-        fileInfo.FullName.Contains
-            $"%c{Path.DirectorySeparatorChar}%s{dirName}%c{Path.DirectorySeparatorChar}"
-    )
+#load "../src/FileConventions/Helpers.fs"
 
 let invalidFiles =
-    Directory.GetFiles(".", "*.yml", SearchOption.AllDirectories)
-    |> Seq.map(fun pathStr -> FileInfo pathStr)
-    |> Seq.filter(NotInDir "node_modules")
-    |> Seq.filter(NotInDir ".git")
-    |> Seq.filter(NotInDir "bin")
-    |> Seq.filter(NotInDir "obj")
-    |> Seq.filter(NotInDir "DummyFiles")
-    |> Seq.filter(fun fileInfo ->
-        FileConventions.DetectUnpinnedVersionsInGitHubCI fileInfo
-    )
+    Helpers.GetInvalidFiles
+        ".."
```

Comment:
@realmarv this is changing from "." to "..", doesn't look right

Reply:
Fixed it